mockyeah/mockyeah
# build: lerna package.json postinstall and contributing docs

## What does it do?

It's possible to run `lerna` right after the contributor does `npm install`. This PR adds that script into `package.json`. But, lets say we have a user that does `npm install --ignore-scripts` but they still want to help. It's no problem! On the Contributing.md docs page it will walk them through.

Bonus: docs site url

## How to test

```sh
git clone -b contributing-lerna-bootstrap https://github.com/youreadforme/mockyeah.git
cd mockyeah
npm install
```

↑ We'll notice right there it also runs lerna. I ran the unit tests locally and they didn't pass but I don't think it's related to this PR. 🤷‍♂
